src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -415,7 +415,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
           language: textTracks[i].language,
           index: i
         };
-        if (settings.language || settings.label) {
```

Comment:
what happens in this case? if 608/708 caption doesn't have a language/label what will be shown?
Do we need a setting option like hls.js expose?

Reply:
also, the test for this now fails